recipe/meta.yaml

```diff
@@ -17,6 +17,7 @@ source:
 build:
   number: {{ build }}
   script: python setup.py install --single-version-externally-managed --record=record.txt
+  noarch: python
```

Comment:
Unfortunately this cannot be `noarch: python` for two reasons. First it uses selectors (e.g. Windows only dependency below). Second it is a dependency of `conda`. As `conda` does all the magic for installing `noarch: python` packages, this is problematic. Have submitted PR ( https://github.com/conda-forge/pysocks-feedstock/pull/6 ) to correct this issue.